app/src/main/res/drawable/ic_arrow_back_white_24dp.xml

```diff
@@ -0,0 +1,10 @@
+<vector
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:width="24dp"
+  android:height="24dp"
+  android:viewportWidth="24.0"
+  android:viewportHeight="24.0">
+  <path
+    android:fillColor="#FFF"
```

Comment:
Hex color should be of 6 characters.

`#FFFFFF`

Reply:
Updated.